receiver/http.go

```diff
@@ -128,19 +132,26 @@ func (htt *HTTP) Start() error {
 		htt.auth = false
 	}
 	for idx, h := range htt.Handlers {
-		log.Printf("Adding handler for %v", idx)
+		log.WithField("handler", idx).Debug("Adding handler")
 		serveMux.Handle(idx, receiver{Handler: h.H, settings: htt})
 	}
-	if htt.Address == "" {
-		log.Printf("HTTP: No listen-address specified. Using go default (probably :http or :https?)")
-	}
+
 	server.Addr = htt.Address
 	if htt.Certfile != "" {
-		log.Printf("Starting http receiver with TLS at %s", htt.Address)
+		log.WithField("address", htt.Address).Info("Starting http receiver with TLS")
 		log.Fatal(server.ListenAndServeTLS(htt.Certfile, htt.Keyfile))
 	} else {
-		log.Printf("Starting INSECURE http receiver (no TLS) at %s", htt.Address)
+		log.WithField("address", htt.Address).Debug("Starting INSECURE http receiver (no TLS)")
 		log.Fatal(server.ListenAndServe())
 	}
 	return skogul.Error{Reason: "Shouldn't reach this"}
 }
+
+// Verify verifies the configuration for the HTTP receiver
+func (htt *HTTP) Verify() error {
+	if htt.Address == "" {
+		log.Warn("Missing listen address for http receiver, using Go default")
+	}
```

Comment:
This is an interesting route to go down that I didn't consider, but which makes sense. If we do this, we should probably eventually move the config-related log-messages in Start()  down here too (e.g.: with/without TLS). But good idea. (also, will be a small merge conflict, but that's not a problem)

Reply:
By "this" I mean: Just warn/log in Verify when things are valid, but potentially breaks POLA.